api/components/icon.schema.json

```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "https://raw.githubusercontent.com/lenra-io/ex_component_schema/beta/priv/static/draft-lenra.json",
+    "$id": "icon.schema.json",
+    "title": "Icon",
+    "description": "The Icon to use",
+    "type": "component",
+    "additionalProperties": false,
+    "required": [
+        "type",
+        "value"
+    ],
+    "properties": {
+        "type": {
+            "description": "The type of the element",
+            "enum": [
+                "icon"
+            ]
+        },
+        "size": {
+            "description": "The size of the Icon",
+            "type": "number",
+            "default": "1"
+        },
+        "color": {
+            "$ref": "defs/color.schema.json",
+            "default": "#000000"
+        },
+        "semanticLabel": {
+            "description": "The semantic label for the Icon.",
+            "type": "string"
```

Comment:
There is no "required" nor "default" for the semanticLabel IMO every property should ether be "required" or have a "default". Also, "semanticLabel" is unclear, could you add a more precise description ? Thx :)